### PR TITLE
LP 130 Basic Generic card creation

### PR DESCRIPTION
**Pull request by juanislas77**
Dashboard - Basic generic card compose component
https://devaptivist.atlassian.net/browse/LP-130

LpGenericCard
image

### CODE REVIEW

**Comment by melissaquezada94** [comment]
Hi Juan, I saw that the component is not equals as the design. In the design all the text are in bold

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 1]
```diff
@@ -32,29 +32,84 @@ import com.iteneum.designsystem.theme.Drab
...
  *
  * @param modifier to modify an specific property of the card
- * @param content as a high order function
+ * @param title as an example 'Current balance'
+ * @param details this is the clickable option
+ * @param accountNumber the showed String number
+ * @param currency if the number represents currency for example current balance
+ * @param onTextClick as a high order function
  *
- * @author Daniel Roldan
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:40]: You can left the original autor and add a new annotation `@modifyBy` and your name

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 2]
```diff
@@ -32,29 +32,84 @@ import com.iteneum.designsystem.theme.Drab
...
-    content: @Composable (ColumnScope.() -> Unit)
+    title: String,
+    details: String,
+    accountNumber: String,
+    currency: Boolean = false,
+    onTextClick: () -> Unit
 ) {
     Card(
         modifier = modifier
             .fillMaxWidth()
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: I recommend you remove this property `.fillMaxWidth()` and send it in the parameter modifier

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 1 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:40]: Hi, comment resolved

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 2 above)
Comment (juanislas77) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Hi, comment resolved

**Review by melissaquezada94 — APPROVED** [review]

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 3]
```diff
@@ -32,29 +32,83 @@ import com.iteneum.designsystem.theme.Drab
...
+    details: String,
+    accountNumber: String,
+    currency: Boolean = false,
+    onTextClick: () -> Unit
 ) {
     Card(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(10.dp),
+        modifier = modifier.padding(10.dp),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Add design size please

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 4]
```diff
@@ -32,29 +32,83 @@ import com.iteneum.designsystem.theme.Drab
...
         ),
         border = BorderStroke(1.dp, MaterialTheme.colorScheme.onPrimary),
         elevation = CardDefaults.cardElevation(1.dp),
-        content = content
     )
+    {
+        Row{
+            Column(modifier = Modifier
+                .fillMaxWidth(0.7f)
+                .padding(all = 15.dp)){
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Add design size please

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 5]
```diff
@@ -32,29 +32,83 @@ import com.iteneum.designsystem.theme.Drab
...
+                    text = details,
+                    color = MaterialTheme.colorScheme.onPrimary,
+                    fontWeight = FontWeight.Medium,
+                    fontSize = 15.sp
+                )
+            }
+            Column(
+                modifier = Modifier
+                .fillMaxWidth()
+                .padding(all = 25.dp),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Add design size please

**Review by carlosph816 — COMMENTED** [review]
Waiting...

**Review by carlosph816 — APPROVED** [review]
Well done!

**Pull request merged**